packages/models-library/src/models_library/rpc_pagination.py

```diff
@@ -74,34 +74,4 @@ def create(
             data=chunk,
         )
 
-    model_config = ConfigDict(
```

Comment:
OK, so as we discussed, we cannot use the examples in generics as before, right?